packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -112,6 +112,7 @@ class PaginatedDataTable extends StatefulWidget {
     this.controller,
     this.primary,
     this.headingRowColor,
+    this.hideEmptyLastPageRows = false,
```

Comment:
Hmm.. naming is hard. Having a negating name default to false can be confusing for users.

What about ..
- showEmptyRows 
- displayEmptyRows
- populateEmptyRows
- includeBlanksRows

I think having a default value of true along with a name to match would be nice. :)